extensions/amp-viewer-integration/amp-doc-viewer-api.md

```diff
@@ -0,0 +1,124 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AMP Viewer/Document API
+
+## Introduction
+
+This document explains the API between an AMP viewer and the AMP document(s) it contains.
+
+## API
+
+### Initialization
+When an AMP viewer opens an AMP document, it can include initialization parameters. The parameters are set as key value pairs encoded as a query string in the hash fragment of the AMP document URL. For example:
+https://cdn.ampproject.org/v/s/www.example.com/article.amp.html?amp_js_v=0.1#origin=https%3A%2F%2Fwww.google.com
+Some parameters are read by the AMP runtime and configure its behavior and others are read by the [viewer integration script](https://github.com/ampproject/amphtml/blob/master/extensions/amp-viewer-integration/0.1/amp-viewer-integration.js) in order to establish the communication channel used by the rest of the API. An initialization parameter is enabled or disabled via the '1' or  '0' value respectively.
```

Comment:
I think we're trying to avoid using the term `AMP runtime`, but I'm not sure if its correct here or not. @pbakaus can you confirm if this should be phrased at the `AMP framework` or...?

Reply:
Well it's technically correct, but it's just not a term commonly used in framework land, which is why I'm trying to avoid it. We are also not explaining it anywhere on amp.dev.

Because it specifically refers to v0.js, we can approve the use here as one-off use because it applies to integrators, but add a sentence descripting what the runtime is, maybe at first use something like "AMP runtime (the core library that initializes and manages all components, included as v0.js)".